### ISSUE TITLE
Add Page Level Offline Data File

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#15]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
Currently, the Offline URL data are available only via Site data. Let's make it available via Page level so that a content directory can be fully modularlized.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
Offline URLs data list is available at page level.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
Offline URLs data list is available at site level only.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Dependency https://github.com/ZORALab/Hestia/issues/16 implemented. Ready to develop.

**Comment by hollowaykeanho** [comment]
Implemented in f31c7b36263f0ccb98660aa56444d1a8e16170e7

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
